rero_ils/modules/api.py

```diff
@@ -629,6 +629,9 @@ def _get_record_class(self, payload):
 
         # take the first defined doc type for finding the class
         pid_type = payload.get('doc_type', 'rec')
+        # correct loan
+        if pid_type == 'loan':
+            pid_type = 'loanid'
```

Comment:
I'm not fan about specific `if` statement into a generic function.... 
I will keep the current existing `LoanIndexer._get_record_class()` function but, of course, returning a correct value.
What's other devs opinion

Reply:
`_get_record_class()` is not working for the task

Reply:
Sorry have not corrected it on the right place.